frontend/packages/local-storage-operator-plugin/src/components/local-volume-set/nodes-selection-list.tsx

```diff
@@ -21,6 +21,7 @@ import {
 import { useSelectList } from '@console/shared/src/hooks/select-list';
 import { hasNoTaints } from '../../utils';
 import { GetRows } from './types';
+import { getZone } from '@console/ceph-storage-plugin/src/utils/install';
```

Comment:
We don't import from the Ceph plugin to LSO plugin that creates a circular dependency. I think you can define a `getZone` function in LSO plugin itself

Reply:
Is this the right path where I can can make this function, instead of importing from ceph plugin??
`console/frontend/packages/local-storage-operator-plugin/src/utils/index.ts`

Reply:
That sounds good to me!

Reply:
okay, thanks.